src/model/specification.ts

```diff
@@ -35,6 +35,18 @@ export type SecuritySubcategory =
   | "Regex"
   | "SSL"
 
+export type ScanType =
```

Comment:
Should use an `enum` (and change the others enumerations while we're at it).

Reply:
is there an advantage to `enum` over the union types?

Reply:
There is an _tabs vs spaces_ kind of debate about those. I'll take types then. 😄